pygameMenu/scrollarea.py

```diff
@@ -297,6 +297,20 @@ def get_view_rect(self):
 
         return rect
 
+    def get_actual_scrollbar_thickness(self, orientation):
+        """
+        Return the scroll thickness of the area. If it's hidden return zero.
+
+        :param orientation: Orientation of the scroll
+        :type orientation: basestring
+        :return: Thickness in px
+        :rtype: int
+        """
+        if orientation == _locals.ORIENTATION_HORIZONTAL and self._world.get_width() > self._rect.width or \

```

Comment:
The condition `self._world.get_width() > self._rect.width`is not enough (there are some specific case when world size is close to area size)

Use `get_hidden_width() == 0` or `get_hidden_height() == 0` to know if scrollbars are displayed or not

Reply:
changed!